communications/newsletters/newsletter_02_Oct2019.md

```diff
@@ -0,0 +1,74 @@
+
+If you live in a country that is not compatible with our [current schedule](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md#attending-an-online-collaboration-cafe) and if you would like to join a call in the future, please add your comment in [this GitHub issue](https://github.com/alan-turing-institute/the-turing-way/issues/711).
+
+## Relevant resources
+
+### Open Life Science Mentoring Program
+
+The Turing Way is delighted to collaborate with the [Open Life Science Program](https://openlifesci.org) organised by Bérénice Batut, Yo Yehudi and Malvika Sharan as one of the [Mozilla Open Leader X projects](https://foundation.mozilla.org/en/opportunity/mozilla-open-leaders/). They are developing this mentoring program for early-stage researchers who want to become ambassadors for Open Life Science in their communities. A call for application for mentees and mentors will be announced soon. Stay up to date through their webpage (https://openlifesci.org/).
+
+## Tips & Tricks for new contributors
+
+Are you new to this project and want to participate in the community? You can start by reading the [contribution guideline](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md), where you can get an overview of the project and the different communication channels you can use to participate.
+
+See [this page](https://github.com/alan-turing-institute/the-turing-way) to find out who is in the community and check out this list of ['good first pull' issues](https://github.com/alan-turing-institute/the-turing-way/labels/good%20first%20issue) to get to know this project well.
+
+## Acknowledgements and celebrations!
+
+Shout out to two of our members [Sarah Gibson](https://twitter.com/drsarahlgibson) and [Rachael Ainsworth](https://twitter.com/rachaelevelyn), who represented The Turing Way project at two separate conferences in September.
+
+Sarah, Research Data Scientist at The Alan Turing Institute, delivered a successful workshop on building a [BinderHub](https://binderhub.readthedocs.io/en/latest/) and The Turing Way at the [RSEconUK 2019](https://rse.ac.uk/conf2019/). She shared her training materials [online on Zenodo](https://zenodo.org/record/3404774#.XZN7h5MzbpB), under the DOI: 10.5281/zenodo.3404774.
+
+Rachael, Community Manager of Software Sustainability Institute, attended the Open Science Fair 2019 in Porto, Portugal, last month, where she presented a poster and a [demo session](https://www.opensciencefair.eu/demos-2019/the-turing-way-a-handbook-for-reproducible-data-science) on The Turing Way project. See her [tweet-thread](https://twitter.com/rachaelevelyn/status/1173542269355532289) from the conference.
```

Comment:
```suggestion
Rachael, Community Manager of Software Sustainability Institute, attended the Open Science Fair 2019 in Porto, Portugal, last month, where she presented a poster and a [demo session](https://www.opensciencefair.eu/demos-2019/the-turing-way-a-handbook-for-reproducible-data-science) on _The Turing Way_ project.
See her [tweet-thread](https://twitter.com/rachaelevelyn/status/1173542269355532289) from the conference, her demo presentation (DOI: [10.5281/zenodo.3403160](https://doi.org/10.5281/zenodo.3403160)) and poster presentation (DOI: [10.5281/zenodo.3381445](https://doi.org/10.5281/zenodo.3381445))!.
```

Reply:
Should we plug the podcast? I think that would be quite nice. https://orionopenscience.podbean.com/e/the-fair-is-in-town-figshare-the-turing-way-and-open-science-quest-at-the-osfair2019/

Reply:
yes. Of course. I missed that.